src/App.js

```diff
@@ -158,7 +158,10 @@ class App extends Component {
             this.fetchFormData();
 
             let formForward;
-            for (let header of response.headers.entries()) {
+            const entries = response.headers.entries();
+            let header;
+            for (let i = 0; i < entries.length; i++) {
```

Comment:
:art: It could be helpful to add a code comment noting that `for` works in IE, `for of` does not.